ui/app/AppLayouts/Chat/views/ChatHeaderContentView.qml

```diff
@@ -44,11 +44,28 @@ Item {
         root.state = d.stateMembersSelectorContent
     }
 
+    function messageAboutToBeSent() {
```

Comment:
WDYT of this?
- Rename `messageAboutToBeSent`. The reasoning here would be that this component doesn't handle messages. There's no reason for it to react on messages. I'd go for a `submitMembersChange`

- This function would do exactly what `MembersEditSelectorView.confirmed` does, right? I think we should use the same flow as on `confirm`. Duplicating the confirm logic could hit us later when it needs to be updated.

All things considering I'd go for something like this:

```
//MembersEditSelectorView

function confirm() { root.confirmed() }
```


```
//ChatHeaderContentView
function submitMembersChange() {
   if(d.selectingMembers)
        membersEditSelectorView.confirm()
}
```